sjtuthesis/build.lua

```diff
@@ -4,13 +4,32 @@ module           = "sjtuthesis"
 
 sourcefiledir    = "source"
 sourcefiles      = {"sjtuthesis.dtx"}
-installfiles     = {"*.sty"}
+installfiles     = {"*.cls","*.def","*.cfg","*.dtx","*.pdf"}
 
--- docfiledir       = "doc"
+docfiledir       = "doc"
 
 typesetexe       = "xelatex"
+typesetfiles     = {"sjtuthesisuserguide.tex","sjtuthesis.dtx"}
```

Comment:
When using `typesetfiles` for typesetting, you probably need to move the `sjtuthesisuserguide.tex` to `doc/` directory for compiling. I can't compile the userguide, at least on Windows.

Reply:
In fact, I can compile this on both Arch Linux and macOS 🤣🤣

I'll investigate this later, let's hold this PR for now.

Reply:
WSL can not pass somehow, maybe l3build doesn't make a good job on Windows.🤔

Reply:
BTW, the artifact from GitHub Actions doesn't have the `userguide` as well. I moved the `.tex` to `doc/` folder, it is at least work on my machine.